redbox-core/redbox/app.py

```diff
@@ -55,14 +55,22 @@ def __init__(
         app.add_node(
             ChatRoute.search,
             get_search_graph(
-                _llm, _parameterised_retriever.with_config(tags=[Redbox.SOURCE_DOCUMENTS_TAG]), _tokeniser, _env, debug
+                _llm,
+                _parameterised_retriever.with_config(tags=[Redbox.SOURCE_DOCUMENTS_TAG]),
+                _tokeniser,
+                env.llm_max_tokens,
```

Comment:
there is a strong case for moving llm_max_tokens to the `AISettings` however this is also used in `get_llm` which is set during the construction of Redbox

Reply:
this makes sense to me. We should be able to get this out of the constructor level to make this work in a follow up PR. More generally I think these kind of vars want to be configurable in the graph as we think about things being more agentic.